src/main/resources/application.conf

```diff
@@ -155,17 +155,21 @@ mantis {
       enabled = true
 
       # Listening address of JSON-RPC HTTP/HTTPS endpoint
-      interface = "127.0.0.1"
+      interface = "localhost"
 
       # Listening port of JSON-RPC HTTP/HTTPS endpoint
       port = 8546
 
-      # Path to the JKS keystore storing the certificate (used only for https)
-      # null value indicates no certificate is being used
+      # Path to the keystore storing the certificates (used only for https)
+      # null value indicates HTTPS is not being used
       certificate-keystore-path = null
 
-      # File with the password used for accessing the certificate (used only for https)
-      # null value indicates no certificate is being used
+      # Type of certificate keystore being used
+      # null value indicates HTTPS is not being used
+      certificate-keystore-type = null
```

Comment:
What are valid keystore types? Maybe that info could be included in the comment here?

Reply:
Detail: a clarification was added to `network.conf`. Could we also add it to this file `application.conf`?